platforms/access-bridge/src/utils.ts

```diff
@@ -3,3 +3,18 @@ export function isValidSiteId(siteId: string): boolean {
   const alphanumericRegex = /^[a-zA-Z0-9]{8}$/;
   return alphanumericRegex.test(siteId);
 }
+
+/**
+ * Checks if the required parameters are present in the provided object.
+ * @param params - The object containing the parameters to be validated.
+ * @param requiredParams - The list of required keys to check for presence.
+ * @returns An array of missing required keys.
+ */
+export const validateBodyParams = <T>(params: Partial<T>, requiredParams: (keyof T)[]): (keyof T)[] => {
```

Comment:
Looks like this is unused? Did you mean to use it in checkout controller?

Reply:
This is forgotten here. It is already removed in one of the following PRs.